src/etc/inc/plugins.inc.d/ipsec.inc

```diff
@@ -1270,7 +1270,7 @@ function ipsec_configure_do($verbose = false, $interface = '')
         set_single_sysctl('net.key.preferred_oldsa', '0');
     }
 
-    $ipseccfg = $config['ipsec'];
+    $ipseccfg = isset($config['ipsec']) ? $config['ipsec'] : [];
```

Comment:
We prefer

```suggestion
    $ipseccfg = $config['ipsec'] ?? [];
```

Reply:
But the question is if we have to go any further if it’s not there/empty.

Reply:
That's neat! I didn't know that shortcut. It's definitely better. I'm not familiar with this file so I'm not sure about other options. Seems like maybe it's trying to do ipsec things even when ipsec isn't configured at all.